cmd/minikube/cmd/root.go

```diff
@@ -145,9 +145,12 @@ func initConfig() {
 	configPath := constants.ConfigFile
 	viper.SetConfigFile(configPath)
 	viper.SetConfigType("json")
-	err := viper.ReadInConfig()
-	if err != nil {
-		glog.Warningf("Error reading config file at %s: %v", configPath, err)
+	if err := viper.ReadInConfig(); err != nil {
+		if _, ok := err.(viper.ConfigFileNotFoundError); ok {
+			glog.Warningf("Config file not found; ignore error if desired")
+		} else {
+			glog.Warningf("Config file was found but another error was produced")
```

Comment:
I know this is still WIP, but for the else case it's important to print the error that was returned. :)

Reply:
Just keep the original message...

Reply:
sure

Reply:
PTAL